test/getOfferApplications.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape');
+const supertest = require('supertest');
+const app = require('../server/app');
+
+// getOfferApplications
+test('test getOfferApplications', (t) => {
+  supertest(app)
+    .get('/api/v1//offer-applications/1')
+    .expect(200)
+    .expect('content-type', /json/)
+    .end((err, response) => {
+      if (err) {
+        t.error(err);
+      }
+      t.deepEqual(
+        Object.keys(response.body),
+        ['error', 'data'],
+        'Should return object of two keys (error and data)',
+      );
+      t.deepEqual(
+        Object.keys(response.body.data[0]),
+        ['member_id', 'username', 'full_name', 'avatar', 'proposal', 'status'],
+        'data should be array of objects each object contains 6 keys',
```

Comment:
@AymanAlqoqa A good improvement you could do to this test, is have it ensure that these data values are also correct.  For example, check that the full_name equals the name that you expect.  Then the test is ensuring that the correct offer and user are being returned, instead of just asserting that any offer and user are being returned.

Reply:
that's right, I'll consider that next time.
thanks 
:+1: